Description: Build system patch
 This patch makes the build system print useful information in case of
 a build failure.
Author: Jeff Licquia <licquia@debian.org>
Forwarded: not-needed
Last-Update: 2011-02-20

Index: htmldoc-1.8.27/Makedefs.in
===================================================================
--- htmldoc-1.8.27.orig/Makedefs.in	2011-02-20 12:18:13.000000000 -0500
+++ htmldoc-1.8.27/Makedefs.in	2011-02-20 12:18:33.000000000 -0500
@@ -81,13 +81,10 @@
 # Rules...
 #
 
-.SILENT:
 .SUFFIXES:	.a .c .cxx .h .o
 .c.o:
-	echo Compiling $<...
 	$(CC) $(CFLAGS) -c $<
 .cxx.o:
-	echo Compiling $<...
 	$(CXX) $(CXXFLAGS) -c $<
 
 #
